docs/spec/staking/state.md

```diff
@@ -6,29 +6,18 @@
  - value: `amino(pool)`
 
 The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, representative
-validator shares for stake in the global pools, moving Atom inflation
-information, etc.
+information about the total amounts of Atoms in all states, moving Atom
+inflation information, etc.
 
 ```golang
 type Pool struct {
     LooseTokens int64   // tokens not associated with any validator
```

Comment:
`LooseTokens` is now tokens not associated with any *bonded* validator, right? (it includes tokens associated with *unbonded validators)

Reply:
Correct - just updated the comment to reflect this